### PR TITLE
perf!: default export to be base videoPlayer

### PR DESCRIPTION
**Pull request by tsi**
BREAKING CHANGE: the default ES import is now only the player core plugins need to be explicitly imported

### CODE REVIEW

File: .npmignore [hunk 1]
```diff
@@ -1,11 +0,0 @@
-!dist/
```
Comment (ehab-cl) [inline .npmignore:1]: why did you delete the npmignore file?

**Review by ehab-cl — APPROVED** [review]
💪

File: .npmignore (hunk 1 above)
Comment (tsi) [reply .npmignore:1]: I wanted to stop pushing our sourcemap files to npm, so I tried doing it via npmignore, then I realized it was redundant ever since we're using the `files` field in package.json
Also [this](https://medium.com/@jdxcode/for-the-love-of-god-dont-use-npmignore-f93c08909d8d) :)
